libs/api/domains/rsk/src/lib/models/currentUserCompanies.model.ts

```diff
@@ -0,0 +1,19 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+@ObjectType()
+export class CurrentUserCompanies {
+  @Field()
+  kennitala!: string
```

Comment:
Translate the fields to English?

Reply:
Yes, not sure why I didn't do this automatically 😅